.trunk/plugins/trunk

```diff
@@ -1 +1 @@
-/Users/gbrown/.cache/trunk/plugins/https---github-com-trunk-io-plugins/v1.4.2-4ebadccd80b22638
\ No newline at end of file
+/Users/gbrown/.cache/trunk/plugins/https---github-com-trunk-io-plugins/v1.6.8-4ebadccd80b22638
```

Comment:
I'm not familiar with the trunk plugin. Is this a file that should be git ignored?